dags/solutions_team/configs/tensorflow/common.py

```diff
@@ -108,10 +108,11 @@ def set_up_tensorflow_2_16_keras() -> Tuple[str]:
 def set_up_google_tensorflow_models() -> Tuple[str]:
   """Common set up for tensorflow models."""
   return (
-      "sudo mkdir -p /usr/share/tpu && cd /usr/share/tpu && git clone https://github.com/tensorflow/models.git",
+      "if [ ! -d \"/usr/share/tpu/models\" ]; then sudo mkdir -p /usr/share/tpu && cd /usr/share/tpu && git clone https://github.com/tensorflow/models.git; fi",
```

Comment:
Should we do a pull if the directory still exists to make sure it's up to date?

Reply:
Pulling it again is causing error `git fatal detected dubious ownership in repository` which occurs because Sourcetree runs git with a different user. This might require extra steps to disable the ownership verification.

Also, I think pulling it again should not be necessary as this is the nightly directory where the models and the image both need to be in sync. When nightly image build, it syncs the same model directory at the same time. So it should be latest and up to date.